modules/pca_address/src/Element/PcaAddress.php

```diff
@@ -57,13 +65,22 @@ public function getInfo() {
    * Process the PCA address form element.
    */
   public static function processPcaAddress(array &$element, FormStateInterface $form_state, array &$complete_form) {
+    // Wrap render array in a wrapper before doing anything.
+    self::wrapAddressFieldsInit($element);
     // Add a lookup text field.
     $element[PcaAddressElement::ADDRESS_LOOKUP] = [
       '#type' => 'textfield',
       '#title' => t('Search Address'),
       '#weight' => -150,
       '#placeholder' => t('Start typing your address'),
     ];
+    // Determine if we need to add a manual input link.
+    if (!isset($element['#allow_manual_input']) || $element['#allow_manual_input'] === TRUE) {
+      $manual_input_link = Link::fromTextAndUrl('Click here', Url::fromUserInput('#enter-address'));
+      $element[PcaAddressElement::ADDRESS_LOOKUP]['#suffix'] = '<div class="manual-address">' . t('@link to enter your address manually.', [
```

Comment:
Please use $this->t().

Reply:
`processPcaAddress()` is a static method, can't call `$this->t()` in non-object mode.